packages/styled-components/src/test/constants.test.js

```diff
@@ -83,17 +83,17 @@ describe('constants', () => {
 
     it('should be true in production NODE_ENV when window.SC_DISABLE_SPEEDY is set to true', () => {
       window.SC_DISABLE_SPEEDY = true;
-      renderAndExpect(true, '.b { color:blue; }');
+      renderAndExpect(true, '.sc-a { color:blue; }');
     });
 
     it('should be true in test NODE_ENV', () => {
       process.env.NODE_ENV = 'test';
-      renderAndExpect(true, '.b { color:blue; }');
+      renderAndExpect(true, '.sc-a { color:blue; }');
     });
 
     it('should be true in development NODE_ENV', () => {
       process.env.NODE_ENV = 'development';
-      renderAndExpect(true, '.b { color:blue; }');
+      renderAndExpect(true, '.sc-a { color:blue; }');
```

Comment:
This is actually a neat change! Every change here is a case where `isStatic` is enabled and effective